<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcitationFrequencyWindow</class>
 <widget class="QDialog" name="ExcitationFrequencyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExcitationFrequencyWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: rgb(214, 219, 233,255);
}

QListView{
background: rgb(214, 219, 233,255);
border: 2px solid rgb(231, 236, 250, 255);
}

QPushButton:hover{
background: rgb(255, 255, 191,255);
border:  2px solid rgb(255, 170, 0,255);
}

QPushButton{
border:  2px solid rgb(231, 236, 250, 255);;
background: rgb(214, 219, 233,255);
}


QHeaderView::section{
background-color: rgb(235, 241, 255, 255);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,4,9,4,4,4,4,4">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetNumEigStates">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_Eigenvalues">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>read eigenvalues</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Eigenvectors">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>read eigenvectors</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonNext">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonPrevious">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonPause">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>pause</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditEigInformation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetAngle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetSpeed">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetSegmentation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonVideoSeq">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save image sequence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>CustomTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonNext</sender>
   <signal>released()</signal>
   <receiver>ExcitationFrequencyWindow</receiver>
   <slot>on_button_next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPrevious</sender>
   <signal>released()</signal>
   <receiver>ExcitationFrequencyWindow</receiver>
   <slot>on_button_previous()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>released()</signal>
   <receiver>ExcitationFrequencyWindow</receiver>
   <slot>stop_visualization()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPause</sender>
   <signal>released()</signal>
   <receiver>ExcitationFrequencyWindow</receiver>
   <slot>on_button_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Eigenvalues</sender>
   <signal>released()</signal>
   <receiver>ExcitationFrequencyWindow</receiver>
   <slot>on_button_eigenvalues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Eigenvectors</sender>
   <signal>released()</signal>
   <receiver>ExcitationFrequencyWindow</receiver>
   <slot>on_button_eigenvectors()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonVideoSeq</sender>
   <signal>released()</signal>
   <receiver>ExcitationFrequencyWindow</receiver>
   <slot>on_button_save_image_sequence()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_next()</slot>
  <slot>on_button_previous()</slot>
  <slot>stop_visualization()</slot>
  <slot>on_button_pause()</slot>
  <slot>on_button_eigenvalues()</slot>
  <slot>on_button_eigenvectors()</slot>
  <slot>on_button_save_image_sequence()</slot>
 </slots>
</ui>
